README.md

```diff
@@ -12,7 +12,7 @@ nodeConfig.addTech(
         comments : true,
         sourcemap : true,
         plugins : [require('cssnext')()],
-        oneOfSourceSuffixes : [['post.css', 'css'], ['ie.post.css', 'ie.css']] // keep just one of found tech. Default is undefined
+        oneOfSourceSuffixes : [['post.css', 'pcss', 'css'], ['ie.post.css', 'ie.css']] // keep just one of found tech. Default is undefined
```

Comment:
Do not insist. On my project remade as necessary. 
Why do I need to add the extension?
Because of IDE support
[postcss support jetbrains](https://plugins.jetbrains.com/plugin/8578-postcss-support)